tests/queries/0_stateless/02270_errors_in_files.sh

```diff
@@ -25,9 +25,10 @@ ${CLICKHOUSE_CLIENT} --query "INSERT INTO TABLE FUNCTION file('test_02270_2.csv.
 ${CLICKHOUSE_CLIENT} --query "SELECT * FROM file('test_02270*.csv.gz', 'CSV', 'a String, b String')" 2>&1 | grep -o -m1 "test_02270_2.csv.gz"
 ${CLICKHOUSE_CLIENT} --query "SELECT * FROM file('test_02270*.csv.gz', 'CSV', 'a String, b String')" --input_format_parallel_parsing 0 2>&1 | grep -o -m1 "test_02270_2.csv.gz"
 
-rm -f "${CLICKHOUSE_TMP}"/test_02270_1.csv
-rm -f "${CLICKHOUSE_TMP}"/test_02270_2.csv
-rm -f "${USER_FILES_PATH}"/test_02270_1.csv
-rm -f "${USER_FILES_PATH}"/test_02270_2.csv
-rm -f "${USER_FILES_PATH}"/test_02270_1.csv.gz
-rm -f "${USER_FILES_PATH}"/test_02270_2.csv.gz
+# remove files
+rm "${CLICKHOUSE_TMP}"/test_02270_1.csv
```

Comment:
btw why change this?

Reply:
@kssenii I feel that this test case not passing is quite strange.  Want to try again.